pkg/skaffold/schema/latest/v1/config.go

```diff
@@ -522,6 +522,9 @@ type DeployConfig struct {
 // for the deploy step. All three deployer types can be used at the same
 // time for hybrid workflows.
 type DeployType struct {
+	// DockerDeploy *alpha* uses the `docker` CLI to create application containers in Docker.
+	DockerDeploy *DockerDeploy `yaml:"docker,omitempty"`
```

Comment:
similar to how the `ko` builder is being implemented, should we start off with hidden properties?
```go
DockerDeploy *DockerDeploy `yaml:"-"`
```

Reply:
whoops sorry i meant to do this. updating

Reply:
should we wait on adding the example project also until the feature is implemented and the field made public?